lib/trino-phoenix5-patched/pom.xml

```diff
@@ -44,10 +44,8 @@
                             <goal>shade</goal>
                         </goals>
                         <configuration>
-                            <createDependencyReducedPom>true</createDependencyReducedPom>
```

Comment:
I would prefer to change it to `false` by default, but I wasn't sure from the documentation if it does not have some other side-effects.